tools/read_data_files/main/main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/m3db/m3db/encoding"
+	"github.com/m3db/m3db/encoding/m3tsz"
+	"github.com/m3db/m3db/persist/fs"
+	"github.com/m3db/m3db/tools"
+	"github.com/m3db/m3x/ident"
+	xlog "github.com/m3db/m3x/log"
+
+	"github.com/pborman/getopt"
```

Comment:
flags not good enough for you eh?

Reply:
meh, this doesn't require an extra var decl so I like it better.